tests/pytest/eligibility/test_views.py

```diff
@@ -71,6 +72,39 @@ def model_EnrollmentFlow_with_form_class(mocker, model_EnrollmentFlow):
     return model_EnrollmentFlow
 
 
+@pytest.mark.django_db
+def test_eligibility_logged_in_filtering_flows(mocker, model_TransitAgency, client):
+    model_TransitAgency.pk = None
+    model_TransitAgency.save()
```

Comment:
For this test, I don't think we need a new transit agency; it's the `EnrollmentFlow`s that we needed to have new instances of. Also, the flows are just using the existing `model_TransitAgency` anyways (`transit_agency=model_TransitAgency,`).

I think we can remove lines 77, 78, 97, and 98.